src/neuro_relay/intercept_proxy.py

```
@@ -144,19 +171,19 @@ async def pump_upstream_to_client():
-            })
+            await self._broadcast(
+                {
+                    "event": "integration_disconnected",
+                    "via": "intercept-proxy",
+                    "details": {"client": str(client_peer)},
+                },
+            )
             print(f"[InterceptProxy] client disconnected {client_peer}")
 
     @staticmethod
     async def _maybe_close(ws):
```

Ruff (0.14.8)

Missing return type annotation for staticmethod `_maybe_close`

Add return type annotation: `None`

(ANN205)